packages/core/src/features/telemetry/renderer/telemetry-white-list-for-functions.injectable.ts

```diff
@@ -91,7 +92,14 @@ const extensions = [
 
 const externalActions = ["open-link-in-browser"];
 
-const uiInteraction = ["show-details"];
+const uiInteraction = [{
+  id: "show-details",
+  getParams: (selfLink: unknown) => {
+    return {
+      kind: parseKubeApi(selfLink as string).resource,
```

Comment:
Is it always expected to be available in the URL? If not:

```suggestion
      kind: parseKubeApi(selfLink as string)?.resource,
```